test-site/gatsby-config.js

```diff
@@ -13,82 +13,7 @@ require(`dotenv`).config({
 
 // this is it's own conditional object so we can run
 // an int test with all default plugin options
-const wpPluginOptions = !process.env.DEFAULT_PLUGIN_OPTIONS
```

Comment:
@TylerBarnes is this what we want to do? so no need for the env var anymore if we're only running it once, right? or is it that we want the former for the schema int suite, and the empty object (formerly w/ cart b/c  woocommerce) for the build int suite?